### ISSUE TITLE
Issue with "How CSS works": (pleonastic phrasing)

### ISSUE THREAD

**Issue by ConsciousClay** [mdn/content#6585]
MDN URL: https://developer.mozilla.org/en-US/docs/Learn/CSS/First_steps/How_CSS_works

#### What information was incorrect, unhelpful, or incomplete?

> Coupled with the way that the cascade works, and the fact that browsers will use the last CSS they come across in a stylesheet when you have two rules with the same specificity you can also offer alternatives for browsers that don't support new CSS.

#### Specific section or headline?
What happens if a browser encounters CSS it doesn't understand?

#### What did you expect to see?
It's not clear how the main clause "_you can also offer alternatives for browsers that don't support new CSS_" follows from the preceding adverbial clauses. How do the cascade and specificity norms of CSS imply that "we can _also_ offer alternatives"? And what is "**also**" implying an addition to?

#### Did you test this? If so, how?
No

<details>
<summary>MDN Content page report details</summary>

* Folder: `en-us/learn/css/first_steps/how_css_works`
* MDN URL: https://developer.mozilla.org/en-US/docs/Learn/CSS/First_steps/How_CSS_works
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/learn/css/first_steps/how_css_works/index.html
* Last commit: https://github.com/mdn/content/commit/c7d9128559ed9526914c9be5bb461445fde5866a
* Document last modified: 2021-02-19T20:09:33.000Z

</details>

**Comment by hamishwillee** [comment]
> Coupled with the way that the cascade works, and the fact that browsers will use the last CSS they come across in a stylesheet when you have two rules with the same specificity you can also offer alternatives for browsers that don't support new CSS.

This is a horrible sentence. What this is saying is that
-  CSS will select the rule with the most *specificity* (i.e.  if you specify to select on `p` elements with an `id` that is more specific than a rule that just says `p` elements. So if you have a rule for both cases, the more specific one will be used where it applies). 
- If you have multiple rules that have the same specificity then CSS will select the last one it finds
- You can couple the above rules + fact that keyworks that aren't understood will be ignored to provide a solution that will work for new features with a fallback to the old ones.

Do you want to take a shot at the text?

**Comment by hamishwillee** [comment]
Let me try again. 

What you are trying to achieve is to define a set of CSS rules, where the most recently defined rules are used where they are supported (e.g. on newer browsers) but there are still "fallback" rules for older browsers that do not understand the new rules.

CSS makes this possible because
- rules are executed in the order they are found - if many rules might apply the last one loaded is used
- more specific rules are applied in preference to less specific rules
- rules that are not understood are just dropped.

The bullet points above allow us to construct rules that achieve the goals outlined in the first paragraph.

**Comment by ConsciousClay** [comment]
Okay, I gave it a shot. I guess the main idea in this sentence was about fallback, right? Because the bulleted considerations had just previously been explained in the article.
